modules/tinymce/rspack.config.js

```diff
@@ -0,0 +1,207 @@
+const path = require("path");
+const fs = require("fs");
+const fg = require("fast-glob");
+const packageData = require("./package.json");
+const { TsCheckerRspackPlugin } = require('ts-checker-rspack-plugin');
+
+const generateDemoIndex = (app) => {
+  const demoList = fg.sync(['src/**/demo/html/*.html', 'src/**/demo/html/**/*.html'], { cwd: __dirname });
+  const sortedDemos = demoList.reduce((acc, link) => {
+    const type = link.split('/')[1];
+
+    if (!acc[type]) {
+      acc[type] = [];
+    }
+
+    acc[type].push(link);
+
+    return acc;
+  }, {});
+
+  const lists = Object.keys(sortedDemos).map(
+    type => `
+    <h2>${type}</h2>
+    <ul>
+      ${sortedDemos[type].map(
+      link => `<li>${type !== 'core' ? `<strong>${link.split('/')[2]}</strong> - ` : ''}<a href="${link}">${path.basename(link)}</a></li>`).join('')
+      }
+    </ul>`
+  ).join('');
+  const html = `
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Demos</title>
+  </head>
+  <body>
+    ${lists}
+  </body>
+  </html>
+  `;
+
+  app.get('/', (_, res) => res.send(html))
+};
+
+const modulesDir = path.resolve(__dirname, "../");
+
+const alias = Object.fromEntries(
+  fs.readdirSync(modulesDir).map((name) => {
+    if (name === 'persona') {
```

Comment:
Could we add a reference comment here to remove this once we flatten away `ephox/<library>` stuff.